app/controllers/catalog_controller.rb

```diff
@@ -234,4 +238,8 @@ class CatalogController < ApplicationController
     # mean") suggestion is offered.
     config.spell_max = 5
   end
+  def admin_state_facet
+    return unless current_user && (current_user.curation_concern_creator? || current_user.admin?)
```

Comment:
Hmm, different problem with `can? :edit, ScannedResource.new`: works for admin/curation_concern_creator, but it results in a 404 for anonymous users.